pipelines/forecast_state.py

```diff
@@ -335,13 +335,31 @@ def main(
             "No data available for the requested report date " f"{report_date}"
         )
 
+    nwss_data_disease_map = {
+        "COVID-19": "covid",
+        "Influenza": "flu",
+    }
+
+    def get_nwss_reports(
```

Comment:
Maybe
```suggestion
    def get_available_nwss_reports(
```

since it just tells you which reports are available.